### ISSUE TITLE
Package from PlayDeb doesn't install on Debian

### ISSUE THREAD

**Issue by tukkek** [ooxi/violetland#138]
It has unmet dependency problems. Trying to install the build dependencies requires an over 300MB download on Debian (probably because of the `libboost-all-dev` package), so that isn't really a good option either.

Please consider fixing the PlayDeb package; submitting an official package to Debian; or studying the possibility of using another distribution mechanism, like AppImage http://appimage.org/

I'd love to package and maintain violetland for Debian but it's currently impossible for me. I'm using Debian Testing (stretch) right now, can't say if it works on Stable (jessie) or Unstable (sid) - but I'd guess not, at least on the last case.

**Comment by ooxi** [comment]
Thank you for your feedback! I'd really love to have violetland as package in Debain, are you a Debian Developer?

While I'm not the maintainer of the playdeb-Package, I will reach out to him in order to provide a proper solution (the currenty playdeb-Package supports Ubuntu only).

I'll try building violetland on stretch and come back to you :-)

**Comment by ooxi** [comment]
The installation instructions for Ubuntu are not sufficient for Debian Stretch. I will fix the `README.md` :-)

Would you mind testing [violetland-debian-stretch.tar.gz](https://github.com/ooxi/violetland/files/903296/violetland-debian-stretch.tar.gz) in the mean time? It's violetland compiled on a clean Debian Stretch installation

**Comment by tukkek** [comment]
> are you a Debian Developer?

No, I'm a debian power-user for decades now, though and in that time I've created my fair share of bug reports and other community actions so I'm pretty familiar with Debian - but not a maintainer or official developer at the moment, no.

> Would you mind testing violetland-debian-stretch.tar.gz in the mean time?

I don't really want to dirty my filesystem with something that isn't simple to remove later on. I know this sounds like it shouldn't take vary long to clean but there are are almost 400 individual files on that package, across several folders and directory hierarchies.

I can compile the game myself if it would help with anything but as I mention before, with the 300MB footprint, it wasn't something I was considering doing just for the sake of trying the newest version of the game. If it's something I can do to help with development/packaging though I'd be glad to do it - unlike the file above, it's very simple for me to remove the packages after I'm done with them.

**Comment by ooxi** [comment]
The linked tar shouldn't taint your filesystem, it's just the compiled game :-)

Nevertheless I'll try installing the current Ubuntu package on Debian Stretch in order to find out which dependency is broken and how to fix it. Moreover you are correct in assuming the `libboost-all-dev` probably isn't necessary and might be replaceable with several smaller dependencies.

That being said thank you very much for your feedback, it's highly appreciated! Since you are familiar with the Debian community maybe you can help me out in how to best get violetland to be packaged for Debian. I tried reading the community guidlines but they are very vague. Are you aware of a program guiding new package maintainers to Debian?

**Comment by tukkek** [comment]
> The linked tar shouldn't taint your filesystem, it's just the compiled game :-)

I've tried running the binary but it complained about missing libsdl2-mixer, even though I have it installed. I figured I would have to put all the folder and files under `/usr/share` or something like that.

> thank you very much for your feedback

No, thank you for Violetland! I played the shit out of the original crimsonland and I was so happy to find your project, years ago! It totally deserves a clone! I haven't played the latest version yet but I had some fun playing it back then for a little while and just came back yesterday trying to figure out what was new!
